src/pages/contributing/translation-program/contributors.tsx

```diff
@@ -51,11 +43,15 @@ export const getStaticProps = (async ({ locale }) => {
   }
 }) satisfies GetStaticProps<BasePageProps>
 
-const Content = (props: BoxProps) => (
-  <Box as={MainArticle} py={4} px={10} w="full" {...props} />
```

Comment:
we need to keep the MainArticle since it is setting an `id` for navigation purposes.